treee111/wahooMapsCreator
# [DEV] move OsmData classes from osm_maps_functions.py to osm_data.py

## This PR…

- refactores `osm_maps_functions.py` so that it is back <1000 lines of code
- pylint back at 10.00/10

## Considerations and implementations

Caused by merging #228 from release-branch to develop via a3856c53802afcffc7a45ec183a783304b6c1643

## How to test

1. run pylint
2. check unittests
3. process a example country/tile

## Pull Request Checklist
    + Especially the [Pull-Requests](https://github.com/treee111/wahooMapsCreator/blob/develop/docs/CONTRIBUTING.md#Pull-Requests) section
